_pages/aizeapaulin.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: aizeapaulin
+pageUrl: aizea-paulin.github.io
+timestamp: T2017-09-24
+---
```

Comment:
I think you need to press enter after the last dash. The formatting makes the CI fail .